sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -171,28 +51,32 @@ class node {
   detail::graph_ptr MGraph;
 };
 
-template <graph_state State = graph_state::modifiable> class command_graph {
+template <graph_state State = graph_state::modifiable>
+class __SYCL_EXPORT command_graph {
```

Comment:
This feels wrong but it seems to work fine. Is it because `State` is not actually used anywhere inside the class?

Reply:
The `__SYCL_EXPORT` feels wrong? I don't think it has anything to do with `State`. Without this attribute `libsycl.so` doesn't have the symbols for the `command_graph<modifiable>` methods, so you get an error from the sycl compiler when trying to build a SYCL application that uses these methods (or just the constructor).